core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js

```diff
@@ -310,11 +321,12 @@ $(document).ready(function() {
       cache : false
     });
 
-  var memberStatsUpdateInterval = setInterval(function() {
-      // todo: need to provision when to stop and start update feature
-      // clearInterval(memberStatsUpdateInterval);
+  $("#myonoffswitch").on( 'change', toggleAutoUpdateSwitch );
 
+  var memberStatsUpdateInterval = setInterval(function() {
+    if(isAutoUpdateTurnedON) {
       loadMemberInfo();
-    }, 5000);
+    }
+  }, 5000);
```

Comment:
What does this 5000 do here?

Reply:
it is a 5 seconds (i.e. 5000 ms) timer for auto refresh.